src/tz390.java

```diff
@@ -1710,7 +1711,7 @@ public  class  tz390 {
      String[]   op_table_370_notsupported = // Table added for RPI 1209A
         {"RIO      S    9C02 D2(B2)",
          "CLRCH    S    9F01 D2(B2)",
-         "CONCS    S    B200 D2(B2)",
+         // DSH #414 new z16 LBEAR "CONCS    S    B200 D2(B2)",
```

Comment:
You've accidentally(?) commented out the entry for CONCS. 
LBEAR does not belong in this table: it never was a 370 instruction
It is correctly defined in op_table_z16.

Reply:
CONCS is being removed because the same opcode is now new z16 LBEAR opcode